<ui version="4.0" >
 <class>encodingDialog</class>
 <widget class="QDialog" name="encodingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Encoding...</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>12</number>
       </property>
       <property name="verticalSpacing" >
        <number>12</number>
       </property>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>190</width>
           <height>0</height>
          </size>
         </property>
         <property name="title" >
          <string/>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Phase:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Video Codec:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="labelVidCodec" >
            <property name="text" >
             <string>None</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="labelPhasis" >
            <property name="text" >
             <string>None</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLabel" name="labelContainer" >
            <property name="text" >
             <string>Unknown</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Container:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="labelAudCodec" >
            <property name="text" >
             <string>None</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Audio Codec:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="title" >
          <string/>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="2" column="1" >
           <widget class="QLabel" name="labelFps" >
            <property name="text" >
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="labelETA" >
            <property name="text" >
             <string>Unknown</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="labelElapsed" >
            <property name="text" >
             <string>00:00:00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_24" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Elapsed:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_26" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Time Remaining:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_28" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Frames/sec:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="title" >
          <string/>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="2" column="1" >
           <widget class="QLabel" name="labelTotalSize" >
            <property name="text" >
             <string>0 MB</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_22" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Total Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="labelAudioSize" >
            <property name="text" >
             <string>0 MB</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_20" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Audio Size:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="labelVideoSize" >
            <property name="text" >
             <string>0 MB</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_18" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Video Size:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="title" >
          <string/>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Processed Frames:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_12" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Total Frames:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="labelFrame" >
            <property name="text" >
             <string>0</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLabel" name="labelVidBitrate" >
            <property name="text" >
             <string>0 kB/s</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_16" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Average Bitrate:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="labelQz" >
            <property name="text" >
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_14" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Quantiser:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="labelTotalFrame" >
            <property name="text" >
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBoxShutdown" >
         <property name="text" >
          <string>Shut down computer when finished</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Priority:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPriority" >
         <property name="currentIndex" >
          <number>2</number>
         </property>
         <item>
          <property name="text" >
           <string>High</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Above Normal</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Below Normal</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Low</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxShutdown</tabstop>
  <tabstop>comboBoxPriority</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
